src/sse/route-service.ts

```diff
@@ -69,20 +69,26 @@ export default class RouteService {
     // TODO handle limit intelligently - wait if we haven't reached that yet, etc
     // this is needed regardless to refresh initial data send to the latest data
     // TODO handle different permanence parameters, eg ?permanence=confirmed or =all 
-    this.sse.updateInit(
-      this.eventService.state.getAllEvents({ limit, minHeight }),
-    );
+    const initialEvent: InitialEvent = {
+      config: {
+        network,
+        type: this.type,
+        id: this.filter,
+        maxConf: confirmationDepth,
+        heartbeat: heartbeatInterval,
+        v:  '0.0.2',
+      },
+      data: this.eventService.state.getAllEvents({ limit, minHeight }),
```

Comment:
What will be the usage of this data? What responsibility will this hold? Just convenience?

Reply:
The use case for chainweb-stream aside from streaming future events is to also get a small slice of historical stuff related to the query

The server will cache results in redis, and if we have any results already in cache when the connection is opened, we ping them over. Before this was under the initial event which is now being deprecated, and config -> .data will host the initial/cached data if it exists

Reply:
Is this data then send over to the server as well? Would it make sense to map over the events and only stream the event id or the like? (not sure if there is one)